tests/assets/xhr-data-url.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2020 New Relic Corporation.
+  PDX-License-Identifier: Apache-2.0
+-->
+<html>
+  <head>
+    <title>RUM Unit Test</title>
+    {init}
+    <script type="text/javascript">
+      // Setting a deny list is necessary or `shouldCollectEvent` is never invoked.
+      NREUM.init||(NREUM.init = {});
```

Comment:
Just FYI you can set the deny list [here](https://github.com/newrelic/newrelic-browser-agent/pull/336/files#diff-0e7b5558d3fda3f5cc9c0c61c4c057e5ae2759b6530c4cbd392f297ee64bd1e5R11) without needing to hardcode it into the test asset

Reply:
Thanks. Once I was done with my reproduction case I could have moved it, but thought it might be good to leave it decoupled from the functional test so the asset itself reproduces the known bug conditions. Maybe fits better in the test though. No problem to move it. Let me know if you think I should.

Reply:
either way all good.  Just wanted to make sure that behavior was clear